src/web/components/elements/DisclaimerBlockComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { body } from '@guardian/src-foundations/typography';
+import { pillarPalette } from '@root/src/lib/pillars';
+import { css } from 'emotion';
+
+const style = (pillar: Pillar) => css`
+    ${body.small()};
+
+    a {
+        color: ${pillarPalette[pillar].dark};
+    }
+
+    sup {
+        font-size: 12.75px;
+    }
+`;
+
+export const DisclaimerBlockComponent: React.FC<{
+    html: string;
+    pillar: Pillar;
+}> = ({ html, pillar }) => (
+    <div
```

Comment:
I'm wondering whether there's a better choice for semantic markup rather than div...

Reply:
Uses <span> in frontend, but it felt easy to use <div> to ensure padding is properly set.

Reply:
Have altered back to span.

Reply:
I couldn't see any wrapping element on frontend, but I think we have to have one here to style the `html`. In which case maybe a `footer` is a reasonable choice?

>The HTML footer element represents a footer for its nearest sectioning content or sectioning root element. A footer typically contains information about the author of the section, copyright data or links to related documents.